LICENSE

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2012 Mavenlink, Inc.
```

Comment:
https://github.com/mavenlink/brainstem/blob/master/LICENSE ...

I mean, thats when it was written, not sure if it changes what date it applies to the thing its attached too...

IANAL